src/test/java/me/karakelley/http/handlers/StaticFilesHandlerTest.java

```diff
@@ -17,13 +16,16 @@
   @Test
   void testGetDisplayFiles() {
     TempFilesHelper.withTempDirectory(directory -> {
-      Path fileOne = TempFilesHelper.createTempFile(directory, "/test1");
```

Comment:
Very nice—any time a test instantiates and names a variable I assume it is referenced somewhere for some reason. I like you took this out here.